arangopipe/tests/container_tests/torch_arangopipe_testcases.py

```diff
@@ -441,12 +463,7 @@ def test_arangopipe_edge_link_add(self):
         return
     
     
-    def tearDown(self):
-        #pass
-        self.admin.delete_arangomldb()
-        self.ap = None
-        self.admin = None
-        return
+
     
 if __name__ == '__main__':
     unittest.main()
```

Comment:
Cannot comment on bin file but do we want to checkin arangopipe/tests/pytorch/.data/ag_news_csv.tar.gz ?

Reply:
Maybe we can just wget all the data bin files? Git is not great for large files